code/mob/living/life/Life.dm

```diff
@@ -274,10 +274,14 @@
 					animate(src, transform = matrix(), time = 1)
 				last_no_gravity = src.no_gravity
 
-			// Zephyr-class interdictor: carbon mobs in range gain a buff to stamina recovery, which can accumulate to linger briefly
-			if (iscarbon(src))
+			//Interdictor's protections for mobs
+			if (isliving(src) && !isintangible(src))
 				for_by_tcl(IX, /obj/machinery/interdictor)
-					if (IX.expend_interdict(4,src,TRUE,ITDR_ZEPHYR))
+					if (IX.expend_interdict(6,src,TRUE)) //This protects mobs from radstorms/wormholes/magnetic biofields
+						src.changeStatus("spatial_protection", 3 SECONDS)
+					if (!iscarbon(src)) //Prevents non-carbons from getting the Zephyr stam boost, but still protects other mobs
+						break
+					if (IX.expend_interdict(4,src,TRUE,ITDR_ZEPHYR)) // Zephyr-class interdictor: carbon mobs in range gain a buff to stamina recovery, which can accumulate to linger briefly
```

Comment:
As there's now an ongoing cost from having people in range at all, the Zephyr cost could probably be reduced, or skipped entirely (making it simply check for mob's carbon status and interdictor's board type inside of the main expend_interdict). Not vital, but worth considering.

Reply:
I could do that, although I'm unsure how costly the interdictors are atm, and having one status effect from interdictors drain battery, and not the other could be a bit confusing